crypto/data_formats.go

```diff
@@ -46,6 +47,9 @@ func HashLogRoot(root trillian.SignedLogRoot) []byte {
 		mapKeyTimestampNanos: strconv.FormatInt(root.TimestampNanos, 10),
 		mapKeyTreeSize:       strconv.FormatInt(root.TreeSize, 10)}
 
-	hash := objecthash.ObjectHash(rootMap)
+	hash, err := objecthash.ObjectHash(rootMap)
+	if err != nil {
+		panic(fmt.Sprintf("ObjectHash(%#v): %v", rootMap, err))
```

Comment:
Are we going to remove this panic in future? If so can you TODO it.

Reply:
This particular function has a static input format. So I think the risk of accidentally triggering a panic after the first run is quite low.  I anyone has strong opinions about it, I can add an error to this return as well.

Reply:
If it does break though the signer will be repeatedly panicing. I think it would be easy to pass the error back as the caller already has an error return. 

If you want to TODO and fix in a followup that's fine to get the build green again.